app/actions/add-new-goal.js

```diff
@@ -1,10 +1,30 @@
-import { INPUT_GOAL } from './../action_types.js';
+import * as types from './../action_types.js';
```

Comment:
@des-des i'm sure you're making a good point, but that literally makes no sense 😉